### PR TITLE
Feature/condo/doma 919/add ticket index mobile screens

### PR DESCRIPTION
**Pull request by Dimitreee**
Screenshot 2021-09-17 at 11 23 05
Screenshot 2021-09-17 at 11 23 10

### CODE REVIEW

File: apps/condo/domains/ticket/components/TicketChanges/index.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import React, { useState } from 'react'
+import { useResponsive } from '../../../common/hooks/useResponsive'
```
Comment (Dimitreee) [inline apps/condo/domains/ticket/components/TicketChanges/index.tsx:2]: remove import

File: apps/condo/domains/ticket/components/TicketChanges/TicketChange.tsx [hunk 2]
```diff
@@ -3,6 +3,7 @@ import { Row, Col, Typography, Tooltip } from 'antd'
 import { has } from 'lodash'
 import styled from '@emotion/styled'
 import { TicketChange as TicketChangeType } from '../../../../schema'
+import { useResponsive } from '../../../common/hooks/useResponsive'
```
Comment (sitozzz) [inline apps/condo/domains/ticket/components/TicketChanges/TicketChange.tsx]:
```suggestion
import { useResponsive } from '@condo/common/hooks/useResponsive'
```

File: apps/condo/domains/ticket/components/TicketChanges/index.tsx (hunk 1 above)
Comment (sitozzz) [inline apps/condo/domains/ticket/components/TicketChanges/index.tsx]:
```suggestion
import { useResponsive } from '@condo/common/hooks/useResponsive'
```

**Review by sitozzz — APPROVED** [review]

File: apps/condo/domains/common/components/Comments/index.tsx [hunk 3]
```diff
@@ -19,10 +20,21 @@ export type TComment = {
     deletedAt: string,
 }
 
-const Container = styled.aside`
+interface IContainerProps {
+    isSmall: boolean
+}
+
+const Container = styled.aside<IContainerProps>`
   background: #F5F5F5;
```
Comment (dkoviazin) [inline apps/condo/domains/common/components/Comments/index.tsx:28]: Maybe, get from colors?

**Review by dkoviazin — APPROVED** [review]

**Review by SavelevMatthew — APPROVED** [review]
LGTM

**Pull request merged**